src/Core_Command.php

```diff
@@ -681,6 +685,22 @@ private function _multisite_convert( $assoc_args ) {
 			} else {
 				WP_CLI::warning( "Multisite constants could not be written to 'wp-config.php'. You may need to add them manually:" . PHP_EOL . $ms_config );
 			}
+		} else {
+			/**
+			 * Multisite constants are defined, therefore we already have an empty site_admins site meta
+			 *
+			 * Code based on parts of delete_network_option
+			 */
+			$rows = $wpdb->get_results( "SELECT meta_id, site_id FROM {$wpdb->sitemeta} WHERE meta_key = 'site_admins'" );
```

Comment:
I'd prefer this to only delete site admin options that are empty (by just adding that as an additional WHERE clause). Otherwise, someone might already have imported a pre-existing DB that contained the proper admin settings, and we are then deleting everything.